stdlib/2and3/distutils/cmd.pyi

```diff
@@ -13,3 +13,28 @@ class Command:
     def finalize_options(self) -> None: ...
     @abstractmethod
     def run(self) -> None: ...
+
+    def announce(self, msg: str, level: int = ...) -> None: ...
+    def debug_print(self, msg: str) -> None: ...
+
+    def ensure_string(self, option: str, default: Optional[str]) -> None: ...
+    def ensure_string_list(self, option: str) -> None: ...
+    def ensure_filename(self, option: str) -> None: ...
+    def ensure_dirname(self, option: str) -> None: ...
+
+    def get_command_name(self) -> str: ...
+    def set_undefined_options(self, src_cmd: str, *option_pairs: Sequence[Tuple[str, str]]) -> None: ...
```

Comment:
This means each entry in `option_pairs` is a sequence of tuples. I haven't checked, but you probably mean `Tuple[str, str]` (each individual arg should be a tuple).

Reply:
The same issue applies in the log.pyi case, I'm sure, ie. `*args` is as below rather than `List[Any]`:
```python
    def log(self, level: int, msg: str, *args: Any) -> None: ...